config/vufind/config.ini

```diff
@@ -601,7 +601,7 @@ username_hint = username_only_letters_numbers_and_basic_punctuation
 ;delete_ratings_with_user = false
 
 ; "Remember me" functionality for listed login methods. If the user chooses to save the login,
-; a login token cookie and a database entry will be created. Users' browser information will be 
+; a login token cookie and a database entry will be created. Users' browser information will be
```

Comment:
My IDE auto formats things like this. Do you prefer not to have this in PRs or does it not bother you? I could deactivate this kind of formatting in my IDE.